wpilibc/src/main/native/include/SampleRobot.h

```diff
@@ -16,7 +16,7 @@ namespace frc {
 class WPI_DEPRECATED(
     "WARNING: While it may look like a good choice to use for your code if "
     "you're inexperienced, don't. Unless you know what you are doing, complex "
-    "code will be much more difficult under this system. Use IterativeRobot or "
+    "code will be much more difficult under this system. Use TimedRobot or "
```

Comment:
Why not `Iterative, TimedRobot, or CommandBased`

Reply:
TimedRobot is better than IterativeRobot in every way except that button triggers in the Command framework don't work right in TimedRobot (see #786).